ddp-workspace/projects/ddp-singular/src/app/components/progress-bar/providers.ts

```diff
@@ -8,11 +8,31 @@ import { ActivityInstance, ActivityStatusCodes, ConfigurationService, UserActivi
 
 
 const getUpdatedActivityMap = (activities: ActivityInstance[], activeActivityId: string): Map<string, ActivityListItem> => {
-  const isConsentSelf = !!activities.find(
-    (activity: ActivityInstance) => activity.activityCode === ActivityCode.ConsentSelf
+  const consentActivityCodes = [ActivityCode.ConsentSelf, ActivityCode.ConsentParental, ActivityCode.ConsentDependent];
+  const consentActivity = activities.find(
+    (activity: ActivityInstance) => consentActivityCodes.includes(activity.activityCode)
   );
```

Comment:
`find()` will return the first one that matches the condition and what if there are multiple consents in an activity?
Is it possible for an activity to have, for example, parental and dependent consent at the same time?

Reply:
Nope, its not possible. There can only be one of these activities at a one time.